acme/acme/crypto_util_test.py

```diff
@@ -69,6 +71,14 @@ def _call(cls, loader, name):
         from acme.crypto_util import _pyopenssl_cert_or_req_san
         return _pyopenssl_cert_or_req_san(loader(name))
 
+    @classmethod
+    def _get_idn_names(cls):
```

Comment:
How was this function determined? There needs to be some kind of inline comment explanation or if it was used to generate the certs, maybe it's better to regenerate those certs during tests rather than dropping static files?